20_classes_lab/01.py

```diff
@@ -1,6 +1,15 @@
-"""
-The following code assumes a class
-named Summer exists. 
-Complete the class so correct result is printed
-"""
-
+#commited by Shlomit
+import sys
+class Summer(object):
+    def __init__(self):
+        self.sum = 0
+
+    def add(self, *list):
+        for n in list:
+            self.sum += int(n)

```

Comment:
Leave one blank line between method definitions in an object. 
See Python's style guide for more:
https://www.python.org/dev/peps/pep-0008/#blank-lines